libc/stdio/putchar.c

```diff
@@ -0,0 +1,7 @@
+#include <io/screen.h>
+#include <stdio.h>
+
+int putchar(int c) {
+  terminal_writechar(c);
```

Comment:
why do you use terminal_writechar and not printf?

Reply:
in the future when this libc will expand into user-space as well, vprintf will call putchar directly instead of terminal_write*. hence, calling printf from putchar will be a circular dependency.